python/cuvs_bench/cuvs_bench/generate_groundtruth/utils.py

```
@@ -40,29 +42,56 @@ def suffix_from_dtype(dtype):
-            shape = (a[0], a[1])
+            final_shape = (n_rows, n_cols)
         else:
-            shape = tuple(
-                [
-                    aval if sval is None else sval
-                    for aval, sval in zip(a, shape)
-                ]
+            header_dims = (n_rows, n_cols)
+            final_shape = tuple(
+                aval if sval is None else sval
+                for aval, sval in zip(header_dims, shape)
```

Ruff (0.15.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)